src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -322,6 +322,19 @@ internal async Task<ImmutableArray<ActiveStatementExceptionRegions>> GetBaseActi
                     continue;
                 }
 
+                // Check if the currently observed document content has changed compared to the base document content.
```

Comment:
I'll admit I don't _quite_ follow this comment...

Reply:
If the current content is equal to baseline content that does not match the debuggee, then that means since baseline was captured the workspace has not observed the change made to the file on disk (there had to be one as the content doesn't match). So, when we are about to apply changes it is ok to ignore this document because the user does not see the change yet in the buffer (if the doc is open) and won't be confused if it is not applied yet. The change will be applied after it's observed by the workspace.

Makes sense?